### PR TITLE
chore: fix some comments

### PR DESCRIPTION
**Pull request by drawdrop**

### CODE REVIEW

**Review by BrendanGlancy — APPROVED** [review]
lgtm

File: src/udp_packets.rs [hunk 1]
```diff
@@ -14,7 +14,7 @@ pub mod udp_payload {
     /// +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
     /// |            Length             |           Checksum            |
     /// +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-    /// We append the data, based on the port the the header
+    /// We append the data, based on the port the header
```
Comment (PsypherPunk) [inline src/udp_packets.rs:17]: Although I agree with the change, does this sentence make sense? 

> _"…based on the port the header"_

Comment (PsypherPunk) [reply src/udp_packets.rs:17]: Ah, should this be "the port **in** the header"?

Comment (BrendanGlancy) [reply src/udp_packets.rs:17]: "We append the data, based on the port"

Comment (PsypherPunk) [reply src/udp_packets.rs:17]: As in that should be the complete sentence?

I think that makes it a bit ambiguous: it's below a diagram with both "Source Port" and "Destination Port"—to which one is it referring? Or neither?

I think it'd be useful to clarify that as part of this, if possible…?

Comment (BrendanGlancy) [reply src/udp_packets.rs:17]: Ahh then it should be "We append the data, based on the destination port"

**Review by PsypherPunk — APPROVED** [review]
Merging, even with the outstanding comment; will raise a PR to fix that as it's technically separate to the changes herein.

**Review by PsypherPunk — CHANGES_REQUESTED** [review]
Scratch that: I didn't see the merge conflict.

@drawdrop, if you can fix the merge conflict, this should be good to merge.

**Comment by bee-san** [comment]
Fixed conflits myself :)

**Pull request merged**